src/emulator/storage/rules/types.ts

```diff
@@ -20,8 +26,10 @@ export interface SourceFile {
 }
 
 export interface RuntimeActionResponse {
-  id: number;
+  id?: number;
+  server_request_id?: number;
```

Comment:
is `server_request_id` dictated by what the JAR returns or something? We don't use underscore-case except in some very limited cases. rulesVersion below, which extends this interface, seems to work - would serverRequestId work here?